docs/auth.md

```diff
@@ -1036,6 +1036,50 @@ authentication cookie.
 The passcode can be sent to the instance's owner via email — more transport
 shall be added later.
 
+### POST /auth/tokens/konnectors/:slug
+
+This endpoint can be used by the flagship application in order to create a
+token for the konnector with the given slug. This token can then be used by the
+client-side konnector to make requests to cozy-stack.
+The flagship app will need to use its own access token to request the konnector 
+token.
+
+#### Request
+
+```http
+POST /auth/tokens/konnectors/impots HTTP/1.1
+Host: cozy.example.org
+Accept: application/json
+Authorization: Bearer eyJpc3Mi...
+```
+
+#### Response
+
+```http
+HTTP/1.1 201 Created
+Content-Type: application/json
+```
+
+```json
+OWY0MjNjMGEtOTNmNi0xMWVjLWIyZGItN2I5YjgwNmRjYzBiCg
```

Comment:
If it is JSON, we should have quotes around the value (`"OW...Cg"`)

Reply:
OK. I wasn't sure there would be quotes around a single string.